vevestaX/vevesta.py

```diff
@@ -387,6 +403,8 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
 
             df_messages.to_excel(writer, sheet_name='messages', index=False)
 
+            self.profilingReport().to_excel(writer, sheet_name='Profiling Report', index=False)
```

Comment:
As asked before...please check if the self.__data is a pandas dataframe...and not empty...only then execute this self.profilingReport()

Reply:
Also, read result of profilingReport into a dataframe and check if the returned dataframe is empty..if not empty write to excel.